frontend/src/components/dashboard/ProcessColumn/index.tsx

```diff
@@ -79,16 +86,25 @@ export default function ProcessColumn({ process, showRejectedApplicant, isPassed
     open();
   };
 
+  const filteredApplicants = useMemo(
+    () =>
+      process.applicants.filter(({ applicantName, isRejected }) => {
+        const matchesName = searchedName ? applicantName.includes(searchedName) : true;
+        const matchesRejection = showRejectedApplicant ? isRejected : !isRejected;
```

Comment:
지금도 잘 동작하는 로직이지만, 아래와 같이 간략화가 가능할 것 같아요.
```suggestion
        const matchesRejection = showRejectedApplicant === isRejected;
```

Reply:
반영했습니다! 꼼꼼하게 봐주셔서 감사해요.☺️